deploy/csi-azuredisk-scheduler-extender.yaml

```diff
@@ -43,7 +43,7 @@ data:
       {"name" : "EqualPriority", "weight" : 1}
     ],
     "extenders" : [{
-      "urlPrefix": "http://localhost:8080/azdiskschedulerextender",
+      "urlPrefix": "http://localhost:8888/azdiskschedulerextender",
```

Comment:
The csi-azuredisk-controller is using 8080 during e2e tests, so I thought it would be better if 8080 is not the default port for the scheduler extender.